src/Microsoft.Fast.Components.FluentUI/wwwroot/DesignTokenInterop.js

```diff
@@ -0,0 +1,123 @@
+﻿import {
+    StandardLuminance,
+    bodyFont,
+    baseHeightMultiplier,
+    baseHorizontalSpacingMultiplier,
+    baseLayerLuminance,
+    controlCornerRadius,
+    density,
+    designUnit,
+    direction,
+    disabledOpacity,
+    strokeWidth,
+    focusStrokeWidth,
+    typeRampBaseFontSize,
+    typeRampBaseLineHeight,
+    typeRampMinus1FontSize,
+    typeRampMinus1LineHeight,
+    typeRampMinus2FontSize,
+    typeRampMinus2LineHeight,
+    typeRampPlus1FontSize,
+    typeRampPlus1LineHeight,
+    typeRampPlus2FontSize,
+    typeRampPlus2LineHeight,
+    typeRampPlus3FontSize,
+    typeRampPlus3LineHeight,
+    typeRampPlus4FontSize,
+    typeRampPlus4LineHeight,
+    typeRampPlus5FontSize,
+    typeRampPlus5LineHeight,
+    typeRampPlus6FontSize,
+    typeRampPlus6LineHeight,
+    accentFillRestDelta,
+    accentFillHoverDelta,
+    accentFillActiveDelta,
+    accentFillFocusDelta,
+    accentForegroundRestDelta,
+    accentForegroundHoverDelta,
+    accentForegroundActiveDelta,
+    accentForegroundFocusDelta,
+    neutralFillRestDelta,
+    neutralFillHoverDelta,
+    neutralFillActiveDelta,
+    neutralFillFocusDelta,
+    neutralFillInputRestDelta,
+    neutralFillInputHoverDelta,
+    neutralFillInputActiveDelta,
+    neutralFillInputFocusDelta,
+    neutralFillStealthRestDelta,
+    neutralFillStealthHoverDelta,
+    neutralFillStealthActiveDelta,
+    neutralFillStealthFocusDelta,
+    neutralFillStrongRestDelta,
+    neutralFillStrongHoverDelta,
+    neutralFillStrongActiveDelta,
+    neutralFillStrongFocusDelta,
+    neutralFillLayerRestDelta,
+    neutralStrokeRestDelta,
+    neutralStrokeHoverDelta,
+    neutralStrokeActiveDelta,
+    neutralStrokeFocusDelta,
+    neutralStrokeDividerRestDelta
+} from 'https://cdn.jsdelivr.net/npm/@fluentui/web-components/dist/web-components.min.js'
```

Comment:
Is there another way to handle this? What happens if someone is using a different CDN or if they are using a local NPM package? Any way to handle that?

Reply:
Yes, this was a bit of a hack to get it working. Need JS expertise to do this differently/better.

Reply:
Hi @EisenbergEffect, It sounds likes @vnbaaij is requesting assistance on this? Any way we can get someone assigned to take this over so we can merge it?  Thanks

Reply:
I've resolved this by allowing a `FluentWebComponentsScriptSource` to be set in the `appsettings.json` file. Working on supplying a default value (the cdn link) if it is not set through the app settings file.